source/getting-started/use-cases/vulnerability-detection.rst

```diff
@@ -63,105 +63,138 @@ Wazuh vulnerability alerts contain relevant information about the identified vul
-       "cluster": {
-         "node": "worker_01",
-         "name": "wazuh1"
-       },
-       "input": {
-         "type": "log"
-       },
-       "agent": {
-         "ip": "10.0.1.64",
-         "name": "Centos",
-         "id": "003"
-       },
-       "manager": {
-         "name": "wazuh-manager-worker-0"
-       },
-       "data": {
-         "vulnerability": {
-           "reference": "https://bugzilla.redhat.com/show_bug.cgi?id=1991687, https://security.gentoo.org/glsa/202209-12, https://security.netapp.com/advisory/ntap-20220930-0001/",
-           "severity": "Medium",
-           "score": {
-             "environmental": "0",
-             "version": "2.0",
-             "temporal": "0",
-             "base": "4.400000"
-           },
-           "cve": "CVE-2021-3697",
-           "package": {
-             "installed": "2021-06-02T06:06:51.000Z",
-             "path": " ",
-             "size": "9264704",
-             "name": "grub2-tools",
-             "description": "Support tools for GRUB.",
-             "type": "rpm",
-             "version": "1:2.02-99.el8",
-             "architecture": "x86_64"
-           },
-           "scanner": {
-             "vendor": "Wazuh"
-           },
-           "description": "A crafted JPEG image may lead the JPEG reader to underflow its data pointer, allowing user-controlled data to be written in heap. To a successful to be performed the attacker needs to perform some triage over the heap layout and craft an image with a malicious format and payload. This vulnerability can lead to data corruption and eventual code execution or secure boot circumvention. This flaw affects grub2 versions prior grub-2.12.",
-           "category": "Packages",
-           "classification": "CVSS",
-           "enumeration": "CVE",
-           "status": "Active"
-         },
-         "aws": {
-           "accountId": "",
-           "region": ""
-         }
-       },
-       "rule": {
-         "firedtimes": 3956,
-         "mail": false,
-         "level": 7,
-         "pci_dss": [
-           "11.2.1",
-           "11.2.3"
-         ],
-         "tsc": [
-           "CC7.1",
-           "CC7.2"
-         ],
-         "description": "CVE-2021-3697 affects grub2-tools",
-         "groups": [
-           "vulnerability-detector"
-         ],
-         "id": "23504",
-         "gdpr": [
-           "IV_35.7.d"
-         ]
-       },
-       "location": "vulnerability-scanner",
-       "decoder": {
-         "name": "json"
-       },
-       "id": "1704377380.22184642",
-       "timestamp": "2024-01-04T14:09:40.138+0000"
-     },
-     "fields": {
-       "timestamp": [
-         "2024-01-04T14:09:40.138Z"
-       ]
-     },
-     "highlight": {
-       "rule.groups": [
-         "@opensearch-dashboards-highlighted-field@vulnerability-detector@/opensearch-dashboards-highlighted-field@"
-       ]
-     },
-     "sort": [
-       1704377380138
-     ]
+    "_index": "wazuh-alerts-4.x-sample-threat-detection",
```

Comment:
**question**: do we want to display the reference to the sample data?

Reply:
I think that it should be no problem, as previously there were images with sample data.